### DISCUSSION TITLE
2022-11-09 Weekly Meeting 81 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#313] [Weekly Meetings]
Weekly Meeting 81 is happening on Wednesday November 9, 2022 from 9:00 AM PST(17:00 UTC) - 10:00 AM PST (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 
- https://github.com/omigroup/omigroup/issues/234 - Audit existing bots
- https://github.com/omigroup/omigroup/issues/289 - Proposed migrations: Jitsi* and ProtonMail/Cal and Element/Matrix

#### Next week: 
TBD!

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
I updated the main README, can we all take a gander at that and confirm I made the proper changes with respect to UTC and the time change here in the U.S.?

Also, could each working group ensure that their READMEs are up to date, and the calendar at omigroup.org is accurately reflecting when you meet?

**Comment by mrmetaverse** [comment]
Hi OMI, could someone fill in for me today? I will be facilitating a workshop offsite in DC and will likely miss the meeting. ♥️

**Comment by madjin** [comment]
Meeting notes graphic:
![omi_meeting](https://user-images.githubusercontent.com/32600939/200907220-625960b3-e8f4-4a33-a451-a7c401455778.jpg)

Not much feedback on demo day theme, personally I think demoing gltf-extensions would be great - @antpb has an awesome demo WIP already that utilizes OMI audio / collider extension.
![brave_d2H1VSEtNT](https://user-images.githubusercontent.com/32600939/200909525-131a3969-9c76-44f5-8b59-2e1d03b97627.png)

There is also an effort to try and get recordings uploaded to YouTube if possible, I think uploading transcriptions would be better (can use OpenAI whisper for example).

Media group is doing a case study for testing Dework, we integrated dework closer into discord, created an early submission form for short content clips: https://app.dework.xyz/omi-group/media-78661/view/board-la8hfnx9?taskId=be50e74e-6bf6-4163-b436-139eb05154ac

Discussed neighborhood with co-working space, suggested that we can build mini booths or displays of the various working groups and perhaps later collab / build off what I've already started with The Street. 
![brave_IXqdhroPTy](https://user-images.githubusercontent.com/32600939/200909476-0348c84a-03f9-4ec4-a75f-d998415c8051.png)
![brave_v2NFClxRFF](https://user-images.githubusercontent.com/32600939/200909484-c8f56d0b-d634-4f9a-adcc-593874dde8dc.jpg)
![brave_emwWnYERI7](https://user-images.githubusercontent.com/32600939/200909496-05ffc13c-394c-4093-b6e0-6a535c0fe1c2.png)
![brave_TSDd6UaGbd](https://user-images.githubusercontent.com/32600939/200909506-bded303c-130c-4472-a171-29397c01ad32.jpg)


> The broader goal with these demos is to show how an OMI neighborhood can be connected on our omigroup.org website in Wordpress
> So everyone here can have expo booth, doors to your own spaces as part of our 3D neighborhood, not just a 2D directory
> coworking space linked above is a central collab zone
> But how we sort out the larger connected XR neighborhood across our sites/spaces is the next big challenge - this is a great v1 of an OMI HQ for meetings using Frame - evo

Call for volunteers to maintain the Mastodon for OMI, seems many colleagues are on the dweb / widerweb mastodon instances

M3 hosting a virtual production showreel / metaverse makers mixer event this Friday 6:30pm

We should do virtual expo conference field trips in December (vket / opensim)
![brave_AmlvvXJGtF](https://user-images.githubusercontent.com/32600939/200909772-c02f7536-f4b1-40b6-8f86-fd5c54ecf8f5.jpg)
![Discord_CAj6oNS8lX](https://user-images.githubusercontent.com/32600939/200909807-935d2bfc-c866-4d37-9e5f-d5f0644c47cd.jpg)
